pkg/probe/wifi_clients.go

```diff
@@ -0,0 +1,92 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func probeWifiClients(c http.FortiHTTP) ([]prometheus.Metric, bool) {
+	var (
+		clientInfo = prometheus.NewDesc(
+			"fortigate_wifi_client_info",
+			"Number of connected access points by status",
+			[]string{"vdom", "mac", "hostname", "wtp_name"}, nil,
+		)
+		clientDataRate = prometheus.NewDesc(
+			"fortigate_wifi_client_data_rate_bit_per_second",
+			"Data rate of the client connection",
+			[]string{"vdom", "mac"}, nil,
+		)
+		wtpBandwidthRx = prometheus.NewDesc(
+			"fortigate_wifi_client_bandwidth_rx_bit_per_second",
+			"Bandwidth for receiving traffic",
+			[]string{"vdom", "mac"}, nil,
+		)
+		wtpBandwidthTx = prometheus.NewDesc(
+			"fortigate_wifi_client_bandwidth_tx_bit_per_second",
+			"Bandwidth for transmitting traffic",
+			[]string{"vdom", "mac"}, nil,
+		)
+		signalStrength = prometheus.NewDesc(
+			"fortigate_wifi_client_signal_strength_decibel",
+			"Signal strength of the connected client",
```

Comment:
I think we should state that this is decibel-miliwatt (dBm) in the description for this and signal_noise below. The fortigate docs says noise is `dB` but that doesn't make sense as far as I can tell, so we can probably assume they mean `dBm`.

Reply:
I'm not sure about that, maybe there is something in the UI about that? I guess you don't have the wifi UI in your setup?

Reply:
No, I don't - I base what I wrote on what I can see in `https://xxxx/api/v2/monitor/wifi`. Which part are you not sure about - that signal strength or the noise?

Reply:
I'm not sure if we can assume the unit of the metrics is `dBm`.   

The troubleshooting guide suggests that both are used. 
https://docs.fortinet.com/document/fortiap/6.4.0/fortiwifi-and-fortiap-configuration-guide/68434/signal-strength-issues

It is not ideal but speaking from this docs I would assume that if the number is positive the unit is `dB` and if it's negative it is probably `dBm`. 

Therefore I will change `_decibel` to `_dBm`.

Reply:
> It is not ideal but speaking from this docs I would assume that if the number is positive the unit is dB and if it's negative it is probably dBm.

Sounds good to me, that should make sense.